modules/graphdb/templates/08_cluster_rejoin.sh.tpl

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# This script focuses on the GraphDB node rejoining the cluster if the scale set spawns a new VM instance with a new volume.
+#
+# It performs the following tasks:
+#   * Rejoins the node to the cluster if raft folder is not found or empty
+
+set -euo pipefail
+
+INSTANCE_ID=$(basename $(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/resourceId?api-version=2021-01-01&format=text"))
+DNS_ZONE_NAME=$(az network private-dns zone list --query "[].name" --output tsv)
+GRAPHDB_ADMIN_PASSWORD="$(az appconfig kv show --name ${app_config_name} --auth-mode login --key graphdb-password | jq -r .value | base64 -d)"
+CURRENT_NODE_NAME=$(cat /tmp/node_name)
+RAFT_DIR="/var/opt/graphdb/node/data/raft"
+NODES=("node-1" "node-2" "node-3")
```

Comment:
I guess we're not going to be scaling up. We can probably construct this based on the `node_count` variable.

Reply:
these scripts do not support scaling up. 
We will also have to change the DNS records and cluster creation ... 
Perhaps in the future or with 10.7 and auto discovery